src/tests/class_tests/openms/source/IsotopeLabelingMDVs_test.cpp

```diff
@@ -0,0 +1,684 @@
+
+  IsotopeLabelingMDVs           isotopelabelingmdvs;
+  OpenMS::Feature               lactate_1_normalized;
+  OpenMS::Feature               lactate_1_with_accuracy_info;
+  OpenMS::FeatureMap            lactate_1_featureMap;
+  OpenMS::FeatureMap            lactate_1_with_accuracy_info_featureMap;
+  
+  // L1_norm_max From CHO_190316_Flux.xlsx provided by Douglas McCloskey
+  // accoa_C23H37N7O17P3S_MRM_measured_13 & fad_C27H32N9O15P2_EPI_measured_48 are extracted from
+  // "MID Max: LC–MS/MS Method for Measuring the Precursor and Product Mass Isotopomer Distributions
+  // of Metabolic Intermediates and Cofactors for Metabolic Flux Analysis Applications, McCloskey et al",
+  // DOI: 10.1021/acs.analchem.5b03887, Supporting Information: Table S-2
+  std::vector<double>           L1_norm_max                             {1.00e+00, 3.324e-05, 2.825e-04, 7.174e-05};
+
+  std::vector<double>           accoa_C23H37N7O17P3S_MRM_theoretical_13 ;
+  std::vector<double>           fad_C27H32N9O15P2_EPI_theoretical_48    ;
+
+  std::vector<double>           accoa_C23H37N7O17P3S_MRM_measured_13    {0.627, 0.253, 0.096, 0.02, 0.004, 0.001};
+  std::vector<double>           fad_C27H32N9O15P2_EPI_measured_48       {0.638, 0.355, 0.1, 0.0, 0.0, 0.0};
+  std::vector<double>           Average_accuracy_groundtruth            {0.02374, 0.03451}; // [accoa_13, fad_48]
+
+  std::vector<OpenMS::Feature>  L1_subordinates_normmax;
+
+  IsotopeDistribution accoa_C23H37N7O17P3S_MRM_theoretical_iso(EmpiricalFormula("C23H37N7O17P3S").getIsotopeDistribution(CoarseIsotopePatternGenerator(6)));
+  for (IsotopeDistribution::ConstIterator it = accoa_C23H37N7O17P3S_MRM_theoretical_iso.begin(); it != accoa_C23H37N7O17P3S_MRM_theoretical_iso.end(); ++it)
+  {
+    accoa_C23H37N7O17P3S_MRM_theoretical_13.push_back( it->getIntensity() );
+  }
+
+  IsotopeDistribution fad_C27H32N9O15P2_EPI_theoretical_iso(EmpiricalFormula("C27H32N9O15P2").getIsotopeDistribution(CoarseIsotopePatternGenerator(6)));
+  for (IsotopeDistribution::ConstIterator it = fad_C27H32N9O15P2_EPI_theoretical_iso.begin(); it != fad_C27H32N9O15P2_EPI_theoretical_iso.end(); ++it)
+  {
+    fad_C27H32N9O15P2_EPI_theoretical_48.push_back( it->getIntensity() );
+  }
+  
+  lactate_1_normalized.setMetaValue("PeptideRef", "Lactate1");
+  for (uint16_t i = 0; i < L1_norm_max.size(); ++i)
+  {
+    OpenMS::Feature sub;
+    sub.setMetaValue("native_id", "Lactate1_"+std::to_string(117+i));
+    sub.setMetaValue("peak_apex_int", L1_norm_max[i]);
+    L1_subordinates_normmax.push_back(sub);
+  }
+  lactate_1_normalized.setSubordinates(L1_subordinates_normmax);
+
+  for (uint8_t i = 0; i < 3; ++i)
+  {
+    lactate_1_featureMap.push_back(lactate_1_normalized);
+  }
+
+  isotopelabelingmdvs.calculateMDVAccuracies(lactate_1_featureMap, lactate_1_with_accuracy_info_featureMap, accoa_C23H37N7O17P3S_MRM_measured_13, "C23H37N7O17P3S");
+  for (uint8_t i = 0; i < lactate_1_with_accuracy_info_featureMap.size(); ++i)
```

Comment:
`uint8_t` is pretty unusual and probably slower than just simply using `size_t`. Also a bit dangerous if the container is larger than 255 entries.